schema-engine/connectors/sql-schema-connector/src/sql_schema_differ/sql_schema_differ_flavour.rs

```diff
@@ -138,6 +138,17 @@ pub(crate) trait SqlSchemaDifferFlavour {
         names.previous == names.next
     }
 
+    /// Check if the given table name is in the given list of tables names.
+    /// The table names  can contain fully qualified table names with namespace
+    /// (e.g. "auth.user") or just the table name.
+    fn contains_table(&self, tables: &[String], namespace: Option<&str>, table_name: &str) -> bool {
+        if let Some(namespace) = namespace {
+            tables.contains(&format!("{namespace}.{table_name}")) || tables.contains(&table_name.to_string())
+        } else {
+            tables.contains(&table_name.to_string())
+        }
```

Comment:
nitpick, but it can be done without the string allocation and also we can extract the shared bit
```suggestion
        tables.iter().any(|t| t == table_name) ||
          namespace.is_some_and(|ns| tables.contains(&format!("{ns}.{table_name}"))
```